Build failure with CRYPTO=mbedtls on Ubuntu 18.04?

Failure happened on Ubuntu 18.04 with cmake version 3.10.2

**Build command:**
`cmake -DARCH=x64 -DTOOLCHAIN=GCC -DTARGET=Release -DCRYPTO=mbedtls ..`

**Errors:**
ENABLE_BINARY_BUILD=0; Building mbedtls library from source.
CMake Error at os_stub/mbedtlslib/mbedtls/library/CMakeLists.txt:232 (install):
  install TARGETS given no ARCHIVE DESTINATION for static library target
  "mbedcrypto".

CMake Error at os_stub/mbedtlslib/mbedtls/library/CMakeLists.txt:232 (install):
  install TARGETS given no ARCHIVE DESTINATION for static library target
  "mbedx509".

CMake Error at os_stub/mbedtlslib/mbedtls/library/CMakeLists.txt:232 (install):
  install TARGETS given no ARCHIVE DESTINATION for static library target
  "mbedtls".

**Does anyone see the same errors?**

It seems that https://github.com/DMTF/libspdm/blob/main/os_stub/mbedtlslib/CMakeLists.txt missed to define LIB_INSTALL_DIR as in 
mbedtls's CMakeLists.txt but I am not sure.

https://github.com/Mbed-TLS/mbedtls/blob/dd79db10014d85b26d11fe57218431f2e5ede6f2/CMakeLists.txt#L254-L257

--- a/os_stub/mbedtlslib/CMakeLists.txt
+++ b/os_stub/mbedtlslib/CMakeLists.txt
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 2.8.12)
 
+if(LIB_INSTALL_DIR)
+else()
+    set(LIB_INSTALL_DIR lib)
+endif()
+
 ADD_SUBDIRECTORY(${CMAKE_CURRENT_LIST_DIR}/mbedtls/library)
 
 # mbedtls

